open-grant-proposals/easyfc.md

```diff
@@ -0,0 +1,126 @@
+# Open Grant Proposal: `EasyFilecoin`
+
+**Name of Project:**`EasyFilecoin`
+
+**Proposal Category:** `app-dev`
+
+**Proposer:** `chainhubdev`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** `Yes`
+
+# Project Description
+
+EasyFilecoin is a cross platform graphical client for Filecoin node. 
+
+The installation and operation of  Filecoin is too complicated for most people. It is difficult to install packages on a Linux or macOS host using the command line and resolve potential software conflicts. Non-professional users are more willing to use graphical software. 
+
+EasyFilecoin would simplify installation, upgrade and configuration of Filecoin node. It provides a clean interface and a series of scripts that interact with Filecoin nodes.
+
+## Value
+
+The Filecoin network requires enough nodes to provide value. EasyFilecoin provides a quick way to participate in the Filecoin network and should speed up the popularity of Filecoin.
+
+As EasyFilecoin is just a graphical client, if the project fails, Filecoin network will not be affected.
+
+The main risks are the technical difficulties and the large workload brought by cross-platform development.
+
+## Deliverables
+
+Graphical client of Filecoin node working on multi platforms which are Linux, macOSOS and Windows. 
+
+Graphical interface of installation and configuration. Users can choose where software will be installed and how much disk space the node uses.
+
+Graphical interface for the creation, backup, and use of Filecoin wallets.
+
+Graphical interface for the Filecoin mining operation including starting, stopping, etc.
```

Comment:
Does this project also include a graphical interface for being a storage client (i.e. importing data, making deals with storage miners, etc)?